libeos-updater-util/tests/checkpoint.c

```diff
@@ -60,6 +59,7 @@ teardown (Fixture       *fixture,
 {
   g_clear_object (&fixture->sysroot);
   g_clear_object (&fixture->repo);
+  g_clear_object (&fixture->root_dir);
```

Comment:
I think this and the `g_unsetenv` below should be rolled into the previous commit.